tests/examples_methods_syntax/scatter_point_paths_hover.py

```diff
@@ -0,0 +1,95 @@
+"""
+Scatter plot with point paths on hover with search box
+======================================================
+This example demonstrates an interactive visualization technique inspired by [this Vega example](https://vega.github.io/vega/examples/global-development/), 
+adapted for Altair's higher-level API. Key features:
+
+1. Visual Hint Paths: Displays trajectories of data items across time on hover, enabling intuitive understanding of how data evolves.
+2. Search Box: Allows filtering of data points by country name, extending the original concept with improved data exploration capabilities
+
+"""
+# category: interactive charts
+
+import altair as alt
+from vega_datasets import data
+
+# Data source
+source = data.gapminder.url
+
+# X-value slider
+x_slider = alt.binding_range(min=1955, max=2005, step=5, name='Year: ')
+x_select = alt.selection_point(name="x_select", fields=['year'], bind=x_slider, value=1980)
+
+# Hover selection
+hover = alt.selection_point(on='mouseover', fields=['country'], empty=False)
```

Comment:
```suggestion
# Hover selection
hover = alt.selection_point(on='mouseover', fields=['country'], empty=False)
# A separate hover for the points since these need empty=True
hover_point_opacity = alt.selection_point(on='mouseover', fields=['country'])
```

I think having the points fade out on hover makes it easier to see the line. See change below for where this is used.